todo_app/templates/index.html

```diff
@@ -13,7 +13,13 @@ <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
         {% for todo_item in todo_items %}
-        <li>{{todo_item.title}}</li>
+        <li>{{todo_item.name}}: {{todo_item.status}}</li>
+        <!-- TODO: Improve button layout -->
+        <form action="http://localhost:5000/complete/{{todo_item.name}}" method="post" class="form-complete">
```

Comment:
This works, but it won't as soon as we host this somewhere other than locally - so referencing just the relative path is likely to be our solution here:
```suggestion
        <form action="/complete/{{todo_item.name}}" method="post" class="form-complete">
```